<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xml_out</class>
 <widget class="QDialog" name="xml_out">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Sections</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="xml_button">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xml_path_line"/>
     </item>
     <item>
      <widget class="QComboBox" name="extension_box">
       <item>
        <property name="text">
         <string>xml</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>xlsx</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Use As</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="use_as_list">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="currentRow">
          <number>-1</number>
         </property>
         <item>
          <property name="text">
           <string>Beam</string>
          </property>
          <property name="icon">
           <iconset resource="../resource_1.qrc">
            <normaloff>:/section/images/beam.svg</normaloff>:/section/images/beam.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Column</string>
          </property>
          <property name="icon">
           <iconset resource="../resource_1.qrc">
            <normaloff>:/section/images/column.svg</normaloff>:/section/images/column.svg</iconset>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ductility</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="ductility_list">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="currentRow">
          <number>-1</number>
         </property>
         <item>
          <property name="text">
           <string>Medium</string>
          </property>
          <property name="icon">
           <iconset resource="../resource_1.qrc">
            <normaloff>:/section/images/r_med.svg</normaloff>:/section/images/r_med.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>High</string>
          </property>
          <property name="icon">
           <iconset resource="../resource_1.qrc">
            <normaloff>:/section/images/r_hight.svg</normaloff>:/section/images/r_hight.svg</iconset>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="shear_section_checkbox">
       <property name="text">
        <string>Shear Sections</string>
       </property>
       <property name="icon">
        <iconset resource="../resource_1.qrc">
         <normaloff>:/section/images/As.svg</normaloff>:/section/images/As.svg</iconset>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource_1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>xml_out</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>xml_out</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
